plugins/pulp_docker/plugins/importers/importer.py

```diff
@@ -126,8 +127,11 @@ def upload_unit(self, repo, type_id, unit_key, metadata, file_path, conduit, con
         :rtype:           dict
         """
         try:
-            upload_step = upload.UploadStep(repo=repo, file_path=file_path, config=config)
+            upload_step = upload.UploadStep(repo=repo, file_path=file_path, config=config,
+                                            metadata=metadata, type_id=type_id)
             upload_step.process_lifecycle()
+        except PulpCodedValidationException as e:
```

Comment:
You could just make this `except PulpCodedValidationException:` and then have L#134 be `raise`